pureport/resource_pureport_google_cloud_connection_test.go

```diff
@@ -94,8 +94,8 @@ resource "pureport_google_cloud_connection" "main" {
   name = "%s"
   speed = "50"
 
-  location_href = "${data.pureport_locations.main.locations.0.href}"
-  network_href = "${data.pureport_networks.main.networks.0.href}"
+  location_href = data.pureport_locations.main.locations.0.href
+  network_href = data.pureport_networks.main.networks.0.href
 
   primary_pairing_key = "${google_compute_interconnect_attachment.main.0.pairing_key}"
```

Comment:
Do you still need the "${...}" formatting here?

Reply:
no, missed that one.

Reply:
Updated in #25